python/tvm/meta_schedule/relay_integration.py

```diff
@@ -384,8 +391,7 @@ def is_meta_schedule_dispatch_enabled() -> bool:
     enabled: bool
         Whether the meta schedule is enabled
     """
-    result = transform.PassContext.current().config.get(
+    return transform.PassContext.current().config.get(
         "relay.backend.use_meta_schedule_dispatch",
-        0,
+        False,
     )
-    return bool(result & 1)
```

Comment:
Fixes in this file are probably necessary after the MS API PR - without them I got a compile error and segfault using `compile_relay`. cc @junrushao